docs/topics/multiplatform/multiplatform-advanced-project-structure.md

```diff
@@ -168,6 +168,36 @@ There are three important concepts in dependency resolution:
 
    ![Error on JVM-specific API in common code](dependency-resolution-error.png){width=700}
 
+### Aligning versions of common dependencies across source sets
```

Comment:
I'm not sure if the new section fits under Dependencies and dependsOn. The existing section describes kinds of dependencies and that's it. so the new one should be a separate section on the second level or be under the Compilations (or closer to it), since it's more concerned with having consistent binaries after the compilation and less with how dependencies are set up